docs/whats-new/preview-5.md

```diff
@@ -0,0 +1,81 @@
+---
+title: .NET Aspire preview 5
+description: .NET Aspire preview 5 is now available and includes many improvements and new capabilities
+ms.date: 03/28/2024
+---
+
+# .NET Aspire preview 5
+
+Breaking Changes
```

Comment:
@eerhardt @mitchdenny can you do this section?

Reply:
Beyond all the packages being split, what other breaking changes were there? I noticed that parameter names changed on the `With*Endpoint` APIs which is breaking, but what else?

Reply:
https://github.com/dotnet/aspire/pulls?q=is%3Apr+label%3Abreaking-change+is%3Aclosed

We aren't marking them properly! 😄 .